app/code/Magento/Widget/Model/Widget/Instance.php

```diff
@@ -34,7 +34,7 @@ class Instance extends \Magento\Framework\Model\AbstractModel
 
     const PRODUCT_LAYOUT_HANDLE = 'catalog_product_view';
 
-    const SINGLE_PRODUCT_LAYOUT_HANLDE = 'catalog_product_view_id_{{ID}}';
+    const SINGLE_PRODUCT_LAYOUT_HANDLE = 'catalog_product_view_id_{{ID}}';
```

Comment:
Unfortunately we cannot rename or remove constants in a PR to `2.2-develop`. I would suggest the best case here would be to add a new constant with the correct spelling and then keep the old constant, deprecate it and then use the new constant to give the old constant it's value. Something like:

```
/**
 * @deprecated see self::SINGLE_PRODUCT_LAYOUT_HANDLE
 */
const SINGLE_PRODUCT_LAYOUT_HANLDE = self::SINGLE_PRODUCT_LAYOUT_HANDLE;

const SINGLE_PRODUCT_LAYOUT_HANDLE = 'catalog_product_view_id_{{ID}}';
```

This way anyone relying on the old constant will still have the constant but all the magento core code that uses it will have been updated to use the new constant.